pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,6 +213,100 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: BuildControllerRoleName,
```

Comment:
@bparees can you take a look at this list of permissions, and let me know if you see anything else the controllers launched from RunBuildController() need? Note that RunBuildPodController and RunBuildImageChangeTriggerController are still using the highly-privileged client.